kafka-ui-api/src/test/java/com/provectus/kafka/ui/container/KafkaConnectContainer.java

```diff
@@ -12,6 +12,7 @@ public class KafkaConnectContainer extends GenericContainer<KafkaConnectContaine
   public KafkaConnectContainer(String version) {
     super("confluentinc/cp-kafka-connect:" + version);
     addExposedPort(CONNECT_PORT);
+
```

Comment:
This empty line could be removed :).

Reply:
I like the spacing like separating logical blocks